app/javascript/components/components/Header.js

```diff
@@ -44,6 +42,9 @@ const Header = (props) => {
               <NavLink to="aboutus">About Us</NavLink>
 
             </NavItem>
+            <NavItem>
+            {logged_in && <NavLink to="/userrecipes">My Saved Recipes</NavLink>}
+            </NavItem>
```

Comment:
```suggestion
            
            {logged_in && <NavItem><NavLink to="/userrecipes">My Saved Recipes</NavLink></NavItem>}
            
```
looks like NavItem shouldn't be rendered if not logged in. Indentation also will likely need fixing